main.go

```diff
@@ -117,6 +117,11 @@ func main() {
 			Usage:  "s3 server",
 			EnvVar: "PLUGIN_SERVER,PLUGIN_ENDPOINT,CACHE_S3_ENDPOINT,CACHE_S3_SERVER,S3_ENDPOINT",
 		},
+		cli.StringFlag{
+			Name:   "accelerated-endpoint",
+			Usage:  "s3 accelerated endpoint",
+			EnvVar: "PLUGIN_ACCELERATED_ENDPOINT,CACHE_S3_ACCELERATED_ENDPOINT,CACHE_S3_ACCELERATED_SERVER,S3_ACCELERATED_ENDPOINT",
```

Comment:
Whats with all the environment variables?

Reply:
I basically copied them from `--server`; which one(s) should I keep?

Reply:
If this would ever be a secret then just keep `CACHE_S3_ACCELERATED_ENDPOINT` and remove the other two.

Reply:
I'm not sure if you would ever want this to be a secret, but I guess if `--server` is a secret then this would be unset or also a secret.

Reply:
Removed the others.